app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] `Like` model should have method that updates the likes counter for a post. This should be similar to the example below:

```ruby
# app/models/like.rb
...

def update_likes_counter
  post.increment!(:likes_counter)
end
```

Reply:
well resolved! ✔️